tests/python/gpu/test_fusion.py

```diff
@@ -238,6 +238,49 @@ def test_fusion_compiler_cache():
     if num_gpus > 1:
         check_fused_symbol(a+b, ctx=mx.gpu(1), a=arr1, b=arr2)
 
+@with_seed()
+def test_fusion_boolean_inputs():
+    from mxnet.gluon import HybridBlock
+    mx.npx.set_np()
```

Comment:
You may use the `@use_np` decorator like in https://github.com/apache/incubator-mxnet/blob/2c02bfffb2bdb4e8f1e5159e08e3250482e5f603/tests/python/unittest/test_numpy_ndarray.py#L28,
https://github.com/apache/incubator-mxnet/blob/2c02bfffb2bdb4e8f1e5159e08e3250482e5f603/tests/python/unittest/test_numpy_ndarray.py#L36-L40